sycl/source/detail/device_impl.cpp

```diff
@@ -284,6 +284,20 @@ bool device_impl::has(aspect Aspect) const {
                MDevice, PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE,
                sizeof(pi_device_type), &device_type,
                &return_size) == PI_SUCCESS;
+  case aspect::ext_intel_device_info_uuid: {
+    auto Result = getPlugin().call_nocheck<detail::PiApiKind::piDeviceGetInfo>(
+        MDevice, PI_DEVICE_INFO_UUID, sizeof(pi_device_type), &device_type,
```

Comment:
Probably I'm missing something, but won't this call write UUID values to the `device_type` variable? If so, why?
Shouldn't 3rg argument be `0` and 4th `nullptr`?

Reply:
here trying to get the size.

Reply:
It seems we should pass 0 and nullptr as 3rd and 4th arguments. Otherwise low level runtime will try to write N bytes to the memory pointed by &device_type.

Reply:
@rbegam Could you please address this comment?

Reply:
did you mean `sizeof(pi_device_type)`, `&device_type` these two parameters? I mistakenly thought `&device_type` and `&return_size`. I think you are right. I am fixing this.